### PR TITLE
feat : added alert and button

### PR DESCRIPTION
**Pull request by MurtazaBagwala07**

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+.fab-icon-holder:hover{
+    opacity: 0.8;
+}
+
+.fab-icon-holder i{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height:100%;
+    color:#ffffff;
```
Comment (romabulani) [inline components/button/button.css]: Dont hardcode color, use variable

File: components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+.fab-label{
+    padding: 2px 5px;
+    align-self: center;
+    user-select: none;
+    white-space: nowrap;
+    border-radius: 0.25rem;
+    font-size:16;
+    background-color:#666666;
+    color: ffffff;
```
Comment (romabulani) [inline components/button/button.css]: here also, hardcoded color, also font- size unit is not present, use rems preferrably

File: components/button/button.css (hunk 1 above)
Comment (MurtazaBagwala07) [reply components/button/button.css]: forgot completely, will change it asap

File: components/button/button.css (hunk 2 above)
Comment (MurtazaBagwala07) [reply components/button/button.css]: > here also, hardcoded color, also font- size unit is not present, use rems preferrably

done

File: components/button/button.css [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    padding:0.25rem;
+}
+
+.fab-label{
+    padding: 2px 5px;
+    align-self: center;
+    user-select: none;
+    white-space: nowrap;
+    border-radius: 0.25rem;
+    font-size:16;
```
Comment (ravikumar1002) [inline components/button/button.css]:
```suggestion
    font-size:1rem;
```

**Comment by ravikumar1002** [comment]
you forgot to add rem or px

File: components/button/button.css (hunk 2 above)
Comment (romabulani) [reply components/button/button.css]: > > here also, hardcoded color, also font- size unit is not present, use rems preferrably
> 
> done

add unit to font-size

Comment (MurtazaBagwala07) [reply components/button/button.css]: done

**Review by romabulani — APPROVED** [review]
Good to merge

**Review by ravikumar1002 — APPROVED** [review]
Looks good, ready to merge

**Pull request merged**